android/src/firebase/core/TitaniumFirebaseCoreModule.java

```diff
@@ -37,9 +39,34 @@ public static void onAppCreate(TiApplication app)
   // Public APIs
   
   @Kroll.method
-	public void configure()
+	public void configure(@Kroll.argument(optional=true)KrollDict param)
 	{
-    // TODO: Allow passing options as well. Right now, it will use the default Firebase-instance
-    FirebaseApp.initializeApp(getActivity().getApplicationContext());
-  }
+		if (param != null) {
+			FirebaseOptions.Builder options = new FirebaseOptions.Builder();
+			if (param.containsKey("APIKey")) {
+				options.setApiKey(param.getString("APIKey"));
+			}
+			if (param.containsKey("databaseUrl")) {
+				options.setDatabaseUrl(param.getString("databaseUrl"));
+			}
+			if (param.containsKey("projectId")) {
+				options.setProjectId(param.getString("projectId"));
+			}
+			if (param.containsKey("storageBucket")) {
+				options.setStorageBucket(param.getString("storageBucket"));
+			}
+			if (param.containsKey("applicationId")) {
```

Comment:
I think it's the same as the `bundleID` used for iOS, so we good get good parity here if you can rename it. From the Firebase Android docs:
```
android {
    defaultConfig {
        applicationId "com.example.myapp"
        minSdkVersion 15
        targetSdkVersion 24
        versionCode 1
        versionName "1.0"
    }
    ...
}
```
So setting the `bundleID` to `Ti.App.id` could set it for both platforms.